src/thoughtspot/thoughtspot-service.ts

```diff
@@ -191,3 +191,27 @@ export async function getDataSources(client: ThoughtSpotRestApi): Promise<DataSo
             }
         });
 }
+
+export async function getDataSourceId(client: ThoughtSpotRestApi, name?: string): Promise<DataSource[]> {
+    const resp = await client.searchMetadata({
```

Comment:
We should pass the `name` to the API and let the backend filter it. Can even do substring match

Reply:
Ack, we do not need this now, will integrate with Kanwar's API instead